firebase-crashlytics/firebase-crashlytics.gradle

```diff
@@ -59,6 +59,7 @@ thirdPartyLicenses {
 }
 
 dependencies {
+    implementation libs.firebase.sessions
```

Comment:
I don't think we should add sessions like this since we depend on it via `implementation(project(':firebase-sessions'))` so we can make changes to sessions and crashlytics today easily.

Reply:
Yeah, android studio did this for some reason and it works without it. I'm switching to vim.